roles/control_node/templates/ansible-navigator.yml.j2

```diff
@@ -5,9 +5,18 @@ ansible-navigator:
     - /home/{{username}}/lab_inventory/hosts
 
   execution-environment:
+{% if workshop_type in ['f5'] %}
+    image: quay.io/acme_corp/f5_modules:latest
+{% endif %}
     enabled: true
     container-engine: podman
     pull-policy: missing
+{% if workshop_type in ['f5'] %}
+    volume-mounts:
+    - src: "/home/{{username}}/ansible_config"
+      dest: "/etc/ansible/"
+      label: "Z"
+{% endif %}
```

Comment:
@anshulbehl would this be necessary of we use default /etc/ansible/ansiblr.cfg location?

Reply:
it won't be, I can change the workshop to use /etc/ansible or we can keep it like this, whatever the team suggests @IPvSean